src/components/AvatarCropModal/Slider.tsx

```diff
@@ -1,34 +1,29 @@
-import PropTypes from 'prop-types';
 import React, {useState} from 'react';
 import {View} from 'react-native';
 import {PanGestureHandler} from 'react-native-gesture-handler';
+import type {GestureEvent, PanGestureHandlerEventPayload} from 'react-native-gesture-handler';
 import Animated, {useAnimatedStyle} from 'react-native-reanimated';
 import Tooltip from '@components/Tooltip';
-import withLocalize, {withLocalizePropTypes} from '@components/withLocalize';
+import useLocalize from '@hooks/useLocalize';
 import useThemeStyles from '@hooks/useThemeStyles';
 import ControlSelection from '@libs/ControlSelection';
-import gestureHandlerPropTypes from './gestureHandlerPropTypes';
+import type {SelectionElement} from '@libs/ControlSelection/types';
 
-const propTypes = {
+type SliderProps = {
     /** React-native-reanimated lib handler which executes when the user is panning slider */
-    onGesture: gestureHandlerPropTypes,
+    onGesture: (event: GestureEvent<PanGestureHandlerEventPayload>) => void;
 
     /** X position of the slider knob */
-    sliderValue: PropTypes.shape({value: PropTypes.number}),
-
-    ...withLocalizePropTypes,
-};
-
-const defaultProps = {
-    onGesture: () => {},
-    sliderValue: {},
+    sliderValue: {
+        value: number;
+    };
```

Comment:
Extract to a separate type and use it here

Reply:
@fabioh8010 For the others, I used `SharedValue<number>` but here we have set the default value `{value: 0}` which we have not done for other instances. Here are the options:
- Use `SharedValue<number>` and remove the default value.
- Use `SharedValue<number>` and set the other props like `addListener`, `removeListener`, `modify` to empty functions.
- Make `{value: number}` into another type and use that with default value.